### ISSUE TITLE
Return database enabled_clients in deterministic order

### ISSUE THREAD

**Issue by lorenh** [auth0/auth0-deploy-cli#275]
### Describe the problem you'd like to have solved

> When we export our tenants to tenants.yaml the enabled_clients list under the database will sometimes get reordered. This makes comparing the version checked in to our source repository to the exported version difficult. Even though no changes occurred, the reordered list shows changes.

### Describe the ideal solution

> Would it be possible to sort the enabled_clients so it can get returned in a deterministic order on export to YAML?

**Comment by jtsampson** [comment]
This looks like is can be accomplished by adding a simple sort to value of `enabled_clients`, in the following files 

1.  `src/context/directory/handlers/databases.js` 
2.  `src/context/directory/handlers/connections.js`2.  
3. ` src/context/yaml/handlers/databases.js`
4.  `src/context/yaml/handlers/connections.js`

```javascript
enabled_clients: [...(database.enabled_clients || []).map(clientId => {
        const found = clients.find(c => c.client_id === clientId);
        if (found) return found.name;
        return clientId;
      })].sort((a, b) => a.toLowerCase().localeCompare(b.toLowerCase())),
```

I tested it locally for json output...would imagine it would work for yaml.

**Comment by lorenh** [comment]
How does this work, will this change eventually be incorporated? Who decides?

**Comment by shushen** [comment]
Fixed by #281
